interface/models.py

```diff
@@ -114,14 +114,7 @@ def state_label(self):
         return self.STATE_CHOICES[self.state]
 
     def evaluate(self):
-        m = re.match(r'https://github.com/(?P<org>[^/]+)/(?P<repo>[^/]+)/?$',
-                     self.assignment.repo_url)
-
-        url_base = ('https://raw.githubusercontent.com/'
-                    '{0}/{1}/'.format(*list(m.groups())))
-
-        config_url = urljoin(url_base,
-                             f'{self.assignment.repo_branch}/checker.sh')
+        config_url = get_config_url(self)
```

Comment:
`script_url = get_script_url(self)` would be better. Please do not forget to update it a few lines bellow too, at `options['env']['script']`